src/cpdag.jl

```diff
@@ -227,3 +227,45 @@ function cpdag(skel::DiGraph)
     end
     g
 end
+"""
+    vskel(g)
+
+Skeleton and `v`-structures. (Currently from the first step of the pc-Alg.)
+"""
+function vskel(g::DiGraph)
```

Comment:
Okay? @mwien

Reply:
Looks good on first glance 👍

Reply:
@RobertGregg this is my current solution

Reply:
That worked for the few examples I tried. Could it be simplified to something like this or am I missing something?:
```julia
for v in vertices(g)
    ns = inneighbors(g, v)

    for (x, y) in combinations(ns, 2) 
        if !isadjacent(g, x, y) 
            remove!(g2, v=>x)
            remove!(g2, v=>y)
        end
    end
end
```

Reply:
(Attempting to) removing an edge requires a binary search, I tried to minimise those a bit by finding out which edges are to be removed